src/test/java/com/exasol/errorcodecrawlermavenplugin/ErrorCodeCrawlerMojoIT.java

```diff
@@ -29,7 +29,7 @@
  * temporary project, runs the plugin on that project and checks the output.
  */
 @Tag("integration")
-//[itest->dsn~mvn-verify-goal~1]
+// [itest->dsn~mvn-verify-goal~1]
```

Comment:
Please see branch [kaklakariada/issue85-ck-refactoring](https://github.com/exasol/error-code-crawler-maven-plugin/tree/kaklakariada/issue85-ck-refactoring) for a proposal about refactoring the integration tests.